client/src/components/SendEmail.js

```diff
@@ -21,7 +21,7 @@ const SendEmail = () => {
         if (email === "" || to === "" || subject === "" || message === "") {
             alert("Please fill all fields");
         }else{
-            const response = await fetch("http://localhost:5000/send", {
+            const response = await fetch("http://localhost:4000/send", {
```

Comment:
I'm just wondering why would we need to change the port here? 

I don't have the .env file so don't know if this is what you've agreed on.